### ISSUE TITLE
Disabled Items reset on world save/close/relaunch.

### ISSUE THREAD

**Issue by Blue001** [CreativeMD/CraftingManager#23]
Not sure what is causing it, but any disabled recipe in your config file, will disappear when the game is closed and reopened.

Added recipes remain correctly however.

It only seems to happen occasionally.

**Comment by RJKoubaka** [comment]
I'm having the same issue. Running 1.0.8 in MC 1.7.10. Using MultiMC as a modpack launching platfor, and forge. Config file not saving anything. All of my self created recipes are saving and carrying over just fine. However, none of my disables are. Not to new worlds, nor to the SAME world when I relog.

**Comment by CreativeMD** [comment]
Tested it, couldn't reproduce this bug, disabled crafting recipes saving just fine. I'm using Forge 10.13.2.1277 and v1.0.8

**Comment by RJKoubaka** [comment]
My forge version is 10.13.2.1264. Perhaps this is the issue?

**Comment by CreativeMD** [comment]
Hm, don't think so but you could try it out.

**Comment by anar4732** [comment]
Please correct this error.
very necessary.
-----------------------------
translated by google.
Original language: Turkish

**Comment by Tsunayoshi** [comment]
I have no problems since v1.0.8. Testet it a week ago, works perfect. SMP, too!

**Comment by anar4732** [comment]
1.0.12 I'm having the same problem.


translated by google.
Original language: Turkish

**Comment by CreativeMD** [comment]
@Tsunayoshi does it also work for 1.0.12?

**Comment by CreativeMD** [comment]
using Cauldron?

**Comment by Tsunayoshi** [comment]
Hey,
sorry I took so long to answer...
I had to drop my old modpack cause hqm messing around with updated mods...

With my new pack I haven't come to the point where I have to disable recipes (so far), but I have testet it now by disabled the forestry bronze ingot recipe (3x copper ingot+1x tin ingot = 4 bronze).

I can confirm it works fine for smp. After relog and after a restart of the server, the recipe is sill disabled!

My Versions:
forge-1.7.10-10.13.2.1352
zCraftingManager ALPHA 1.0.16 mc1.7.10
